arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -623,6 +672,50 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/clearAlerts": {
+      "post": {
+        "tags": [
+          "Alerts"
+        ],
+        "description": "Clear the alerts.",
+        "operationId": "Alerts_Clear",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The clear alert request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClearAlertRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully cleared the alerts."
+          }
+        },
+        "deprecated": false
```

Comment:
I'm not sure how Swashbuckle decides whether to write the property or not, it's ok to leave it there as it doesn't make a difference, I just thought I'd let you know. thanks!